donate/app.py

```diff
@@ -29,6 +30,11 @@ def create_app(config_object=ProdConfig):
 
     app.get_stripe_key = get_stripe_key
 
+    # Handle metrics requests.
+    @app.route("/metrics")
+    def metrics():  
+        return Response(prometheus_client.generate_latest(), mimetype=prometheus_client.CONTENT_TYPE_LATEST)
```

Comment:
Where is Response supposed to come from?  failing to find it

Reply:
Flask thing I forgot to add.